draft-ietf-tsvwg-careful-resume.xml

```diff
@@ -1058,6 +1035,41 @@ Observing ...> Connect -> Reconnaissance --------------------> Normal
         
     </section> <!-- Validating Phase -->
 
+    <section anchor="sec-notes-BBR" title="Implementation Notes for using BBR">
+
+        <t>When the flow is controlled using BBR, Careful Resume is implemented
+           by setting the pacing rate from the saved congestion control parameters,
+           with the following precautions:</t>
+           <t><list style="symbols">
+		   <t>The flag "carefully-resuming" is added to the BBR state
+              to indicate that the sender is allowed to send unvalidated
+		      packets.  
+		       It is initialized to "False" when the BBR flow starts;</t>
+		   <t>Careful Resume is allowed to transmit unvalidated packets
+			  only when the BBR flow is in the Startup state;</t>
+			  <t> The probing rate is configured to 1/2 of the bottleneck
+			bandwidth, as derived from the CWND calculation specified in the
+			saved congestion control parameters according to the
+			  requirements in <xref target="sec-phase-unv-phase"></xref>;</t>
+		  <t>The sender starts the Unvalidated Phase at the beginning of a round,
+			 and sets the "carefully-resuming" flags to "True";</t>
+			 <t>When the "carefully-resuming" flag is set, the BBR congestion 
+			 controller sets the BBR pacing rate
+			 to the larger of the nominal pacing rate (BBR.bw multiplied bytes
+			 BBRStartupPacingGain)
+			 or the calculated probing rate. Then, CWND is set to the larger of BBR.bw
+			 and the probing rate, multiplied by BBR.rtt_min times
+			 BBRStartupCwndGain;</t>
+		  <t>The "carefully-resuming" flag is reset to False two rounds after
+			 it is set, i.e., after all the packets sent in the first round
+			 of "carefully resuming"
+			 have been received and acknowledged by the peer.
+			At that stage (after the capacity has been validated),
+			 the measured delivery rate is expected to reflect the probing rate.</t>
```

Comment:
It would be good to also say something about if not validated (i.e. a cross-reference to the safe-resuming).

Reply:
Could this be as simple as: " Section <XREF to SafeRetreat> describes how the sender adapts the rate after congestion is experienced when sending unvalidated packets".